### PR TITLE
doc: convert to ReST documentation

### PR DESCRIPTION
**Pull request by acopar**
setup.py, widgets/**init**.py: Add help path
doc: add make and conf.py files for sphinx
doc: convert markdown to ReST format and fix structure
.gitignore: ignore documentation build dir

### CODE REVIEW

File: doc/widgets/chaining.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+Chaining
+========
+
+.. figure:: icons/chaining.png
+   :alt: Chaining widget icon
+
+   Chaining widget icon
```
Comment (kernc) [inline doc/widgets/chaining.rst:7]: This is printed.

File: doc/widgets/chaining.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+through chaining of latent matrices along paths in a data fusion graph.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Fitted Fusion Graph**
+
+Fitted collective latent data model.
```
Comment (kernc) [inline doc/widgets/chaining.rst:18]: This dedentation breaks the list. See how it renders: https://github.com/biolab/orange3-datafusion/pull/19/files?short_path=cfbfdec#diff-cfbfdec10a0c528d24b648cd66e2b2f2

**Comment by ajdapretnar** [comment]
@kernc This was fixed in a subsequent PR, if you'd care to check.

**Comment by kernc** [comment]
Right, thanks. Sorry, I'm a bit behind.

**Pull request merged**